cli/internal/app/install.go

```diff
@@ -0,0 +1,97 @@
+package app
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/speakeasy-api/gram/cli/internal/app/logging"
+	"github.com/speakeasy-api/gram/cli/internal/flags"
+	"github.com/speakeasy-api/gram/cli/internal/mcp"
+	"github.com/speakeasy-api/gram/cli/internal/profile"
+	"github.com/speakeasy-api/gram/cli/internal/workflow"
+	"github.com/urfave/cli/v2"
+)
+
+var installFlags = []cli.Flag{
+	flags.APIKey(),
+	flags.Project(),
+	&cli.StringFlag{
+		Name:  "toolset",
+		Usage: "The slug of the Gram toolset to install (e.g., speakeasy-admin). Will automatically look up MCP configuration.",
+	},
+	&cli.StringFlag{
+		Name:  "mcp-url",
+		Usage: "The MCP server URL (e.g., https://mcp.getgram.ai/org/project/environment). Use this for manual configuration.",
+	},
+	&cli.StringFlag{
+		Name:  "name",
+		Usage: "The name to use for this MCP server in Cursor (defaults to toolset name or derived from URL)",
+	},
+	&cli.StringFlag{
+		Name:  "header-name",
+		Usage: "The HTTP header name for the API key (defaults to Gram-Apikey)",
+		Value: "Gram-Apikey",
+	},
```

Comment:
- `header-name` -> `header`
- `Gram-Apikey` does not seem right. Shouldn't it be `Authorization`?